src/main/webapp/fileManagement.jsp

```diff
@@ -13,189 +13,196 @@
 <%@page import="org.apache.commons.fileupload.servlet.*" isThreadSafe="false" %>
 <%@page import="org.apache.commons.fileupload.util.*" isThreadSafe="false" %>
 <%@include file="functions.jsp" %>
-<!DOCTYPE html>
-<html>
-    <head>
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-        <%
+<%@ include file="configuration.jsp" %>
+
+
+    <%
+        int benutzerID = ((Integer) session.getAttribute("BenutzerID")).intValue();
+        Benutzer benutzer = BenutzerDB.getById(benutzerID);
+        boolean isAdmin = benutzer.isAdmin();
+
+        if (isAdmin) {
```

Comment:
Same here, this should be checked in the servlet.

Reply:
done look at commit: isAdminRequired function